src/multirust-cli/setup_mode.rs

```diff
@@ -21,13 +21,20 @@ pub fn main() -> Result<()> {
              .help("Enable verbose output"))
         .arg(Arg::with_name("no-prompt")
              .short("y")
-             .help("Disable confirmation prompt."));
+             .help("Disable confirmation prompt."))
+        .arg(Arg::with_name("default-toolchain")
+             .short("d")
```

Comment:
I guess the other option we're likely to add here soon is something like `--no-modify-path`. Not sure how that will affect short flags.